pkgs/development/python-modules/netcdf4/default.nix

```diff
@@ -1,50 +1,73 @@
 { stdenv, buildPythonPackage, fetchPypi, isPyPy, pytest
-, numpy, zlib, netcdf, hdf5, curl, libjpeg, cython, cftime
-}:
-buildPythonPackage rec {
-  pname = "netCDF4";
-  version = "1.5.1.2";
+, numpy, zlib, netcdf, hdf5, curl, libjpeg, cython, cftime 
+,mpi4py ? null, openssh ? null}:
```

Comment:
space

Reply:
I overlooked this request. So I removed the extra indentation and rebased again.
I have to admit that I am not too familiar with the pull requests workflow, so I apologize for any confusion I might have created by going about it in a roundabout way.

Reply:
Is there more to be done?